members/B001288.yaml

```diff
@@ -134,6 +134,8 @@ contact_form:
     - click_on:
       - value: Submit
         selector: button[type='submit'][class='btn']
+    - wait:
+      - value: 5
```

Comment:
is it possible to use a `find` instead? also the `find` after this has a very weak selector, can that be improved?

Reply:
Updated. I edited the selector to be `#form-2A9B34FE-A96D-4B35-5C05-91EF928C204A div p`